### PR TITLE
Wayf 67 add search stub

### PR DESCRIPTION
**Pull request by TylerMaloney**
# Description

This PR includes the following proposed change(s):

WAYF-65-add-search-stub

cap2

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Updating Testing Framework(s)
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

### CODE REVIEW

File: src/frontend-pwa/src/App.tsx [hunk 1]
```diff
@@ -2,12 +2,14 @@ import './App.css';
...
 import Footer from './components/Footer/Footer';
 import ViewRouter from './routes/ViewRouter';
+import SearchBar from './components/SearchBar/SearchBar';
 
 function App() {
   return (
     <div>
       <Header />
       <ViewRouter />
+      <SearchBar />
```
Comment (LocalNewsTV) [inline src/frontend-pwa/src/App.tsx:12]:
```suggestion
```

File: src/frontend-pwa/src/App.tsx [hunk 2]
```diff
@@ -2,12 +2,14 @@ import './App.css';
 import Header from './components/Header/Header';
 import Footer from './components/Footer/Footer';
 import ViewRouter from './routes/ViewRouter';
+import SearchBar from './components/SearchBar/SearchBar';
```
Comment (LocalNewsTV) [inline src/frontend-pwa/src/App.tsx:5]:
```suggestion
```

File: src/frontend-pwa/src/components/SearchBar/SearchBar.tsx
Comment (LocalNewsTV) [inline src/frontend-pwa/src/components/SearchBar/SearchBar.tsx:1]: Should be a reusable component like the button

**Review by LocalNewsTV — CHANGES_REQUESTED** [review]

File: src/frontend-pwa/src/components/SearchBar/SearchBar.tsx
Comment (Bottle7) [inline src/frontend-pwa/src/components/SearchBar/SearchBar.tsx:1]: Make sure to add accessible labels to all your user input elements. The `label` element is preferred for this; however, the properties `aria-labelledby` and `aria-label` are also acceptable.

File: src/frontend-pwa/src/components/SearchBar/searchbar.styles.ts
Comment (Bottle7) [inline src/frontend-pwa/src/components/SearchBar/searchbar.styles.ts:1]: Avoid using `px` values, whenever possible. The majority of the time `em` / `rem` will be the units you use, depending on your use case.

File: src/frontend-pwa/src/components/SearchBar/searchbar.styles.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    display: inline-block;
+`;
+
+export const StyledInput = styled.input`
+  ${typography.toString()}
+  max-width: 800px;
+  width: 80%;
+  padding: 2px 50px 2px 5px;
+  border-radius: 10px;
+  font-size: 15px;
```
Comment (Bottle7) [inline src/frontend-pwa/src/components/SearchBar/searchbar.styles.ts:19]: Don't mess around too much with `font-size` and other globally set properties. Typically, UI/UX will provide a very strict typographical guide—body text is usually `font-size: 16pt`, as an example—there are very few times where you will be overriding something like this.

Comment (TylerMaloney) [reply src/frontend-pwa/src/components/SearchBar/searchbar.styles.ts:19]: duly noted! was trying to get a "stub" up, and perhaps ran roughshod over the guide in doing do. will refine going forward.

**Review by LocalNewsTV — APPROVED** [review]

**Pull request merged**